package.json

```diff
@@ -32,6 +32,7 @@
     "lint:all": "npm run lint --workspaces --if-present",
     "lint:fix:all": "npm run lint --workspaces --if-present -- --fix",
     "load": "(cd utils && npm run load)",
+    "prepare": "husky",
```

Comment:
This does not replace any existing commits when run (runs automatically on `npm i`), at least when I've tested this